plugins/modules/rds_global_cluster_info.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023 Ansible Project
+# Copyright (c) 2023 Gomathi Selvi Srinivasan (@GomathiselviS)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+module: rds_global_cluster_info
+version_added: 6.6.0
+short_description: Obtain information about Aurora global database clusters
+description:
+  - Obtain information about Aurora global database clusters.
+options:
+    global_cluster_identifier:
+        description:
+          - The user-supplied Global DB cluster identifier.
+          - If this parameter is specified, information from only the specific DB cluster is returned.
+          - This parameter is not case-sensitive.
+          - If supplied, must match an existing DBClusterIdentifier.
+        type: str
+    filters:
```

Comment:
According to documentation here (https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DescribeGlobalClusters.html) this parameter isn't currently supported, this can be removed and the modules restricted to a lister of cluster id